flipt-evaluation/src/lib.rs

```diff
@@ -161,15 +162,30 @@ where
         self.store = store_impl;
     }
 
+    pub fn list_flags(&self) -> ListFlagsResult {
+        let _r_lock = self.mtx.read().unwrap();
+        let flags = match self.store.list_flags(&self.namespace) {
+            Some(f) => f,
+            None => {
+                return Err(Error::Unknown(format!(
+                    "failed to get flags for {}",
+                    self.namespace,
+                )));
+            }
+        };
+
+        Ok(flags)
```

Comment:
Take it or leave it (im also guessing based on what I think I remember):
```suggestion
        match self.store.list_flags(&self.namespace) {
            Some(f) => Ok(f),
            None => Err(Error::Unknown(format!(
                "failed to get flags for {}",
                self.namespace,
            ))),
        };
```